ui/app/mainui/AppMain.qml

```diff
@@ -886,8 +894,17 @@ Item {
 
                                         emojiPopup: statusEmojiPopup
                                         stickersPopup: statusStickersPopupLoader.item
-
-                                        rootStore: appMain.rootChatStore
+                                        sectionItemModel: model
+
+                                        rootStore: ChatStores.RootStore {
+                                            contactsStore: appMain.rootStore.contactStore
+                                            emojiReactionsModel: appMain.rootStore.emojiReactionsModel
+                                            openCreateChat: createChatView.opened
+                                            chatCommunitySectionModule: {
+                                                appMain.rootStore.mainModuleInst.prepareCommunitySectionModuleForCommunityId(model.id)
+                                                return appMain.rootStore.mainModuleInst.getCommunitySectionModule()
```

Comment:
I wonder if this will cause issue. Once we call it for another community, I think it might garbage collect the previous one. I hope I'm wrong because it's a very annoying issue it fix. We need to test to see if after switching communities a couple of times it, the community functionalities still work when accessing `chatCommunitySectionModule`

Reply:
It is created for each ChatLayout (community section)
So each section will have its own `rootStore` with `chatCommunitySectionModule`
But yeh, testing is required 💯